Cython/Build/Cythonize.py

```diff
@@ -45,10 +45,12 @@ def find_package_base(path):
         package_path = '%s/%s' % (parent, package_path)
     return base_dir, package_path
 
-
 def cython_compile(path_pattern, options):
-    pool = None
     all_paths = map(os.path.abspath, extended_iglob(path_pattern))
+    _cython_compile(all_paths, options)
```

Comment:
This should probably read `_cython_compile_paths`. But I'm actually not sure that `_cython_compile_paths` is a good name. "Paths" sounds too ambiguous. Maybe `_cython_compile_files` ?

Reply:
> This should probably read _cython_compile_paths

Ah :-/ I am sorry for such stupid error. Fixed and renamed.